doc/iterations/2023.8/shapes/order_and_filter_from_pages.adoc

```diff
@@ -0,0 +1,40 @@
+= (S) Add an API for filtering and ordering view properties pages
+
+== Problem
+
+A form in view properties can be composed of several pages from various contributions.
+A specifier should be able to order and filter these pages to control the form representation.
+
+== Key Result
+
+Offer the specifier the ability to programmatically order and filter the pages of a form.
```

Comment:
Isn't it already possible with the current version of the tool?
Pages have domain type and precondition expression, so it is already possible to filter the pages.
Pages can be moved with the FormDescriptionEditor so it is already possible to order the pages.

Reply:
I think the idea is for *applications* (which did not necessarily define the Forms definitions and this can not change their preconditions) to control "after the fact" which pages they want to see and in which order.

Reply:
If my interpretation is correct, "A specifier should be able to order and filter these pages to control the form representation." should probably be reformulated. Indeed, if we assume this will be used by the specifier (of the form), it is redundant with the existing mechanism.
The proposed API makes sense however for applications which reuse existing Forms but want some control (maybe exposed to their end-users) on the Pages to display.